spec/quickstart/quickstart-cli.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * Created by krzysztof on 01.02.16.
+ */
+
+'use strict';
+
+var logger = require('log4js').getLogger('CLI');
+var SSH = require('simple-ssh');
+var fs = require('fs');
+
+process.chdir('build/vagrant-quickstart');
+
+describe('CLI test', function () {
+
+    var deploymentName = 'nodecellar';
+    var stdOptions = {
+        out: function (result) {
+            logger.trace('Out: ' + result);
+        }, exit: function (code, out, err) {
+            logger.trace('Exit: ' + code + ' ' + out + 'err: ' + err);
+        }
+    };
+    var ssh;
+
+    // no angular. so ignoreSynchro
+    browser.ignoreSynchronization = true;
+
+    function createSSH(baseDir) {
+        return new SSH(
+            {
+                host: '127.0.0.1',
+                user: 'vagrant',
+                port: 2222,
+                key: fs.readFileSync('.vagrant/machines/default/virtualbox/private_key'),
+                baseDir: baseDir
+            }
+        );
+    }
+
```

Comment:
All of the above code is in the global scope of the 'describe', you should put it inside a beforeEach or beforeAll block.